Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM scratch
-COPY ./cacert.pem /etc/ssl/certs/ca-certificates.crt
-COPY cardigann-linux-amd64 /cardigann
+FROM alpine:latest
+
+ENV CARDIGANN_PKG cardigann-stable-linux-amd64.tgz
```

Comment:
Also, this one should probably be an ARG, so that is can be externally customized.